feature/manager/build.gradle

```diff
@@ -0,0 +1,40 @@
+plugins {
+    id "com.android.library"
+    id "kotlin-android"
+    id "kotlin-kapt"
+    id "app.cash.paparazzi"
+}
+
+apply from: rootProject.file("buildScripts/common.gradle")
+apply from: rootProject.file("buildScripts/common-compose.gradle")
+
+dependencies {
+    implementation project(":core")
+    testImplementation project(":test-utils")
+
+    implementation Libs.hilt_android
+    kapt Libs.hilt_android_compiler
+    kapt Libs.hilt_compiler
+
+    implementation Libs.ui
+    implementation Libs.ui_tooling
+    implementation Libs.ui_tooling_preview
+    implementation Libs.foundation
+    implementation Libs.androidx_compose_material_material
+    implementation Libs.material_icons_extended
+    implementation Libs.lifecycle_viewmodel_compose
+
+    implementation Libs.showkase
+    kapt Libs.showkase_processor
+
+    // FIX: added these 2 due to ClassNotFoundExceptions that affect only preview. Remove if/when fixed
```

Comment:
Is there a link to an issue worth pasting here?

Reply:
stackoverflow link I guess. Have to search it again though